src/test/java/seedu/address/testutil/EditModuleDescriptorBuilder.java

```diff
@@ -0,0 +1,77 @@
+package seedu.address.testutil;
+
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import seedu.address.logic.commands.module.EditModuleCommand.EditModuleDescriptor;
+import seedu.address.model.module.Module;
+import seedu.address.model.module.ModuleCode;
+import seedu.address.model.module.ModuleDescription;
+import seedu.address.model.module.ModuleName;
+import seedu.address.model.tag.Tag;
+
+/**
+ * A utility class to help with building EditPersonDescriptor objects.
```

Comment:
Should this be EditModuleDescriptor instead?

Reply:
Fixed, thanks! 👍